docs/rules/object-curly-newline.md

```diff
@@ -7,34 +7,26 @@ A number of style guides require or disallow line breaks inside of object braces
 ## Rule Details
 
 This rule enforces consistent line breaks inside braces.
-This rule is applied to both object literals and destructuring assignments.
 
 ## Options
 
-```json
-{
-    "object-curly-newline": ["error", {"multiline": true}]
-}
-```
-
-This rule has options of 4 kinds:
+This rule has a string option:
 
-* `"always"` - requires line breaks always.
-* `"never"` - disallows line breaks.
-* `{multiline: true}` (default) - requires line breaks if there are line breaks inside properties or between properties. Otherwise, disallows line breaks.
-* `{minProperties: <integer>}` - requires line breaks if the number of properties is more than the given integer. Otherwise, disallows line breaks.
+* `"always"` - requires line breaks inside braces
+* `"never"` - disallows line breaks inside braces
 
-`multiline` and `minProperties` can be combined.
+This rule has an object option:
 
-* `{multiline: true, minProperties: <integer>}` - requires line breaks if there are line breaks inside properties or between properties, or if the number of properties is more than the given integer. Otherwise, disallows line breaks.
+* `"multiline": true` (default) - requires line breaks if there are line breaks inside properties or between properties. Otherwise, disallows line breaks.
+* `"minProperties"` - requires line breaks if the number of properties is more than the given integer. Otherwise, disallows line breaks.
 
 Also, we can separate configuration for each object literal and destructuring assignment:
```

Comment:
You can specify different options for object literals and destructuring assignments: